### PR TITLE
SEARCH-621 (New exe for checking disk space)

### PR DESCRIPTION
**Pull request by keerthi16**
## Description
Included new FreeDiskSpace utils [electron-utils](https://github.com/symphonyoss/electron-utils).
[SEARCH-621](https://perzoinc.atlassian.net/browse/SEARCH-621)

## Approach
How does this change address the problem?
- #### Problem with the code: Permission issue checking disk space on windows 7 
- #### Fix: Included new FreeDiskSpace utils.

## Learning
N/A

#### Blog Posts
N/A

## Open Questions if any and Todos
- [x] Unit-Tests
[Test Results — Spectron.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1815139/Test.Results.Spectron.pdf)
[Test Results — Unit Tests.pdf](https://github.com/symphonyoss/SymphonyElectron/files/1815140/Test.Results.Unit.Tests.pdf)
- [x] Documentation
- [ ] Automation-Tests

@VikasShashidhar Please review. Thanks

### CODE REVIEW

**Comment by VikasShashidhar** [comment]
@keerthi16 Please squash your commits. Thanks.

File: js/search/searchUtils.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ class SearchUtils {
         return new Promise((resolve, reject) => {
             if (!isMac) {
                 try {
-                    this.path = this.path.substring(0, 2);
+                    this.path = this.path.substring(0, 1);
```
Comment (VishwasShashidhar) [inline js/search/searchUtils.js]: Please add code comments explaining what this does.

File: js/search/utils/checkDiskSpace.js [hunk 2]
```diff
@@ -25,25 +27,25 @@ function checkDiskSpace(path, resolve, reject) {
             return resolve(space >= searchConfig.MINIMUM_DISK_SPACE);
```
Comment (VishwasShashidhar) [inline js/search/utils/checkDiskSpace.js]: Please add code comments explaining what this does.

File: js/search/utils/checkDiskSpace.js [hunk 3]
```diff
@@ -25,25 +27,25 @@ function checkDiskSpace(path, resolve, reject) {
...
-                }
-                return reject(new Error("Error : " + error));
+                log.send(logLevels.ERROR, `Error retrieving free disk space : ${error}`);
+                log.send(logLevels.ERROR, `Error stderr: ${stderr}`);
             }
-            if (stdout.indexOf(searchConfig.PERMISSION_ERROR) !== -1) {
-                // this is temporary until we use the custom exe file.
-                return resolve(true);
+
+            let data = stdout.trim().split(",");
```
Comment (VishwasShashidhar) [inline js/search/utils/checkDiskSpace.js]: Please add code comments explaining what this does.

File: js/search/searchUtils.js (hunk 1 above)
Comment (keerthi16) [reply js/search/searchUtils.js]: Included in the code documentation

**Pull request merged**